meson.build

```diff
@@ -2657,48 +2643,56 @@ endforeach
 
 ############################################################
 
-prev = ''
-foreach p : fuzz_regression_tests
-        a = p.split('/')[-3]
-        b = p.split('/')[-2]
-        c = p.split('/')[-1]
+# Enable tests for all supported sanitizers
+foreach tuple : sanitizers
```

Comment:
Would it make sense to run
```sh
$ python infra/helper.py build_fuzzers --sanitizer <address/memory/undefined> $PROJECT_NAME <source_path>
$ python infra/helper.py reproduce $PROJECT_NAME <fuzz_target_name> <testcase_path>
```
to reproduce bugs found by `OSS-Fuzz`? This way seems to be more robust.

Reply:
The way that oss-fuzz runs tests is very inefficient: it downloads images of all tools and libraries and needs docker, etc. In contrast to this, the regression testing in systemd only requires asan to be installed locally and rebuilds systemd twice.

... and I think it makes a lot of sense to run those tests in two very different ways: once the way that oss-fuzz is doing that on google servers, and once the way we are doing it our own way.

Reply:
I agree that the oss-fuzz way isn't very efficient, but it seems to be effective, which I think is more helpful. Anyway, it has almost nothing to do with the PR and could be discussed later if anyone would be bothered :-)

Reply:
It turned out that it is currently not possible to run the two commands I mentioned in scripts looking for regressions because of https://github.com/google/oss-fuzz/issues/1246.